gcloud/storage/test_batch.py

```diff
@@ -362,6 +354,16 @@ def test_finish_nonempty_non_multipart_response(self):
         batch._requests.append(('DELETE', URL, {}, None))
         self.assertRaises(ValueError, batch.finish)
 
+    def test_current(self):
+        from gcloud.storage.batch import _BATCHES
+        klass = self._getTargetClass()
+        batch_top = object()
+        self.assertEqual(list(_BATCHES), [])
+        _BATCHES.push(batch_top)
+        self.assertTrue(klass.current() is batch_top)
+        _BATCHES.pop()
+        self.assertEqual(list(_BATCHES), [])
```

Comment:
This test seems unrelated to the PR (a merge / rebase artefact?).

Given that it belongs here, it should probably test that `klass.current()` returns `None` before and after the `push()`/`pop()`.